tests/Unit/ConfigMgrCBDsc.ReverseDsc.tests.ps1

```diff
@@ -3111,6 +3111,206 @@ InModuleScope $script:subModuleName {
                         }
                     )
                 }
+                @{
+                    ImplementedAs = 'PowerShell'
+                    Name          = 'CMSiteConfiguration'
+                    ModuleName    = 'ConfigMgrCBDsc'
+                    Version       = '1.0.1'
+                    Properties    = @(
+                        @{
+                            Name         = 'SiteCode'
+                            PropertyType = '[string]'
+                            IsMandatory  = $true
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'Comment'
+                            PropertyType = '[string]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'ClientComputerCommunicationType'
+                            PropertyType = '[string]'
+                            IsMandatory  = $false
+                            Values       = '{HttpsOnly,HttpsOrHttp}'
+                        }
+                        @{
+                            Name         = 'ClientCheckCertificateRevocationListForSiteSystem'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'UsePkiClientCertificate'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'UseSmsGeneratedCert'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'RequireSigning'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'RequireSha256'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'UseEncryption'
+                            PropertyType = '[bool]'
+                            IsMandatory  = $false
+                            Values       = '{}'
+                        }
+                        @{
+                            Name         = 'MaximumConcurrentSendingForAllSite'
+                            PropertyType = '[UInt32]'
```

Comment:
i know this is throughout, but this should be lower case like string and bool should be uint32 in this section